tox.ini

```diff
@@ -59,4 +59,4 @@ exclude=.venv,.git,.tox,dist,doc,*lib/python*,*egg,build
 [pytest]
 # pytest configuration
 # see: http://docs.pytest.org/en/latest/customize.html#adding-default-options
-# addopts = -v
+# addopts = -v
```

Comment:
here as well

Reply:
Can do, but the original file did not have a newline at the end. Should I add it anyway?

Reply:
its showing that you made a change here, if you add one, this change should go away,